jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@8.2.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+            timeout(time: 1, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
+            registryUrl 'https://public.ecr.aws/'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'RELEASE_VERSION',
+            description: "Required: On-going release version, e.g. 2.0.0",
+            trim: true
+        )
+        string(
+            name: 'RELEASE_CHORE',
+            description: "Required: What release chore you wanna do? e.g: 'add_rc_details_comment'",
+            trim: true
+        )
+    }
+    stages {
+        stage('Parameter check') {
+            steps {
+                script {
+                    if (!(RELEASE_VERSION && RELEASE_CHORE)) {
```

Comment:
Can you just do RELEASE_VERSION without params?

Reply:
And if it is only one entry for now do we even need this parameter?

Reply:
RELEASE_CHORE will have multiple entires like release_notes check and notify, check entrance criteria, etc. Hence added check for both.

Reply:
I think just `RELEASE_VERSION` should be working without `params` prefix.

Reply:
Added params so that it handles the error efficiently rather than throwing:
```
Also:   hudson.remoting.ProxyException: org.jenkinsci.plugins.workflow.actions.ErrorAction$ErrorId: 11744410-4cd5-4f4e-b825-8453d841b329
hudson.remoting.ProxyException: groovy.lang.MissingPropertyException: No such property: RELEASE_VERSION for class: WorkflowScript
```